{"cveId": "CVE-2018-12557", "cwe": ["CWE-200", "CWE-200"], "cvss": [{"baseScore": "5.0", "version": "2.0", "Attack Vector": "None", "Attack Complexity": "Low", "Privileges Required": "None", "User Interaction": "None", "Scope": "None", "Confidentiality Impact": "Partial", "Integrity Impact": "None", "Availability Impact": "None"}], "references": ["http://lists.zuul-ci.org/pipermail/zuul-announce/2018-June/000015.html", "https://storyboard.openstack.org/#!/story/2002177", "https://git.zuul-ci.org/cgit/zuul/commit/?id=ffe7278c08e6e36bf8b18f732c764e00ff51551e"], "description": ["An issue was discovered in Zuul 3.x before 3.1.0. If nodes become offline during the build, the no_log attribute of a task is ignored. If the unreachable error occurred in a task used with a loop variable (e.g., with_items), the contents of the loop items would be printed in the console. This could lead to accidentally leaking credentials or secrets."], "published": "2018-06-19T05:29:00.230", "state": "PUBLIC", "vendorName": ["n/a"], "productName": ["n/a"], "github": {"advisories": [], "commits": [], "pocAdvisorie": null, "repo": null, "info": {}}, "pocList": []}